code/def_files/main-f.sdr

```diff
@@ -251,12 +247,9 @@ vec3 CalculateLighting(vec3 normal, vec3 diffuseMaterial, vec3 specularMaterial,
    vec3 lightSpecular = vec3(0.0, 0.0, 0.0);
 	#pragma optionNV unroll all
 	for (int i = 0; i < MAX_LIGHTS; ++i) {
-#if __VERSION__ > 120
 		if (i > n_lights)
 			break;
-#endif
-		if(i > 0)
```

Comment:
Yeah, I don't think those lines should have been removed. This basically will apply shadows meant for the scene directional light onto point lights and tube lights in forward lighting passes.